_src/themes/mit/static/sass/_variables.scss

```diff
@@ -0,0 +1,47 @@
+// ----------------------------
```

Comment:
@clrux, sorry for the late-breaking comment, but one thing I just noticed is that this file/directory structure doesn't match the one we planned for in https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#main-style-compile. Can you sync up where this is (meaning move it to the ``_utilities.scss`` file of the MIT theme)?